src/controllers/developer.ts

```diff
@@ -245,3 +245,21 @@ export const downloadAllDatasetFiles = async (req: Request, res: Response, next:
     next(new NotFoundException('errors.import_missing'));
   }
 };
+
+export const rebuildCube = async (req: Request, res: Response, next: NextFunction) => {
+  const dataset = singleLangDataset(res.locals.dataset, req.language);
+  const revision = dataset.draft_revision;
+  if (!revision) {
+    logger.error('No draft revision found');
+    next(new NotFoundException('errors.draft_missing'));
+    return;
+  }
+
```

Comment:
This route does an `Include.All` but only uses the draft revision id. Hopefully not a regularly used route but just something to keep in mind for other routes.